dist/libblockdev.spec.in

```diff
@@ -58,6 +50,23 @@ Requires: glib2-devel
 This package contains header files and pkg-config files needed for development
 with the libblockdev library.
 
+%package -n python2-%{name}
+Summary:     Python2 gobject-introspection bindings for libblockdev
+Requires: %{name}%{?_isa} = %{version}-%{release}
```

Comment:
I think we should also Provide: the ``python-`` package here. Or use ``python-`` and Provide: ``python2-``.